civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -5567,9 +5567,10 @@ function _user_redirection() {
     $query = $current_path ? sprintf('?destination=%s', $requestURI) : '';
     $redirect_path = 'welcome-page' . $query;
   }
-  elseif (in_array($current_path, ['', 'civicrm', 'civicrm/dashboard'])) {
-    $tasksDashboard = 'civicrm/tasksassignments/dashboard#/tasks';
-    $redirect_path = user_access('access CiviCRM') ? $tasksDashboard : 'dashboard';
+  elseif (in_array($current_path, ['', 'civicrm', 'civicrm/dashboard']) ||
```

Comment:
Can we split the check into multiple lines for [readability](https://www.drupal.org/docs/develop/standards/coding-standards#linelength) if its ok by you? 🤷‍♂️ .

```
$pathExist = in_array($current_path, ['', 'civicrm', 'civicrm/dashboard']);
$isAnonymous = !user_is_anonymous();
$isWelcomePage = drupal_get_destination()['destination'] == 'welcome-page';
....
elseif ($pathExist || $isAnonymous && $isWelcomePage){
...
}
```

Reply:
I initially did it this way but the only issue I have with it is that the variables will be evaluated at all times even when the elseif condition does not get called. What do you think?

Reply:
I think the space/ time complexity is still negligible. I will suggest probably going with what you did earlier as it will help anyone picking it up later to update make sense of it easily.